test/apiTest.js

```diff
@@ -0,0 +1,52 @@
+// next we will write an API test for GET by id, DEL by id, POST and PUT routes: 
+
+const request = require('supertest');
+const app = require('../src/app');
+
+//==================== user API test ====================
+
+/**
+ * Testing get all user endpoint
+ */
+describe('GET /cats', function () {
```

Comment:
These unit tests look like a great start 🎉!  It might feel like overkill and slow to write so many, but they really become valuable as a project grows larger and help you write better code :)